.github/workflows/static.yml

```diff
@@ -0,0 +1,52 @@
+# Simple workflow for deploying static content to GitHub Pages
+name: Deploy static content to Pages
+
+on:
+  pull_request:
+    branches: ['main']
+  push:
+    branches: ['main']
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+concurrency:
+  group: 'pages'
+  cancel-in-progress: false
+
+jobs:
+  deploy:
+    environment:
+      name: github-pages
```

Comment:
Given this deployment does that mean it's regularly accessible? Meaning could I just use this as an import (not saying I will just curious options wise)

Reply:
I see no issue why we can import js from here directly but i think it might be safer for applications to maintain their own copy

Reply:
Copy that, was just curious